Gruntfile.js

```diff
@@ -87,7 +87,11 @@ module.exports = function(grunt) {
         options: {
           jshintrc: '.jshintrc'
         },
-        src: 'src/js/**/*.js'
+        src: [
+          'src/js/**/*.js',
+          '!src/js/extensions/jquery.js',
+          '!src/js/extensions/jquery.once.js'
```

Comment:
They should also probably be separated out into the respective library names since they're separate projects entirely (e.g. `lib/jquery/jquery.js` and `lib/jquery.once/jquery.once.js`).